packages/mux-player/src/helpers.ts

```diff
@@ -16,20 +16,19 @@ export const getPlayerVersion = () => player_version;
 
 export const getSrcFromPlaybackId = (
   playbackId?: string,
-  { token, domain = MUX_VIDEO_DOMAIN }: { token?: string; domain?: string } = {}
+  { maxResolution, token, domain = MUX_VIDEO_DOMAIN }: { maxResolution?: string; token?: string; domain?: string } = {}
 ) => {
   /*
-   * 2022-04-01 djhaveri
+   * 2023-02-11 djhaveri
```

Comment:
can we use `git blame` for this? 

if we all start adding our username / date it's gonna get noisy